react/features/lobby/middleware.js

```diff
@@ -311,6 +313,8 @@ function _conferenceFailed({ dispatch, getState }, next, action) {
                 descriptionKey: 'lobby.joinRejectedMessage'
             }, NOTIFICATION_TIMEOUT_TYPE.LONG)
         );
+
+        setTimeout(() => dispatch(cancelKnocking()), NOTIFICATION_TIMEOUT.LONG);
```

Comment:
I think we should instead use an alert dialog with an ok button which performs the cancelKnocking upon click.